nevergrad/benchmark/experiments.py

```diff
@@ -128,22 +143,27 @@ def yawidebbob(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
 
 
 @registry.register
-def wide_discrete(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+def instrum_discrete(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     # Discrete, unordered.
     optims = ["DiscreteOnePlusOne", "Shiwa", "CMA", "PSO", "TwoPointsDE", "DE", "OnePlusOne",
-              "CMandAS2", "PortfolioDiscreteOnePlusOne"]
+              "CMandAS2", "PortfolioDiscreteOnePlusOne", "DoubleFastGADiscreteOnePlusOne"]
 
     seedg = create_seed_generator(seed)
-    for nv in [10, 50, 200]:
+    for nv in [10, 50, 200, 1000, 5000]:
```

Comment:
This does not make it faster at all :D

Reply:
Thanks to your changes below maybe it will work :-)